src/phase_panel.py

```diff
@@ -214,7 +214,7 @@ def update_data(self, output):
                         inRange *= energy <= self.GetPlotParam('E_max')
                 elif self.GetPlotParam('set_E_max'):
                     inRange = energy <= self.GetPlotParam('E_max')
-                inRange *= ~nan_ind
+                inRange *= not nan_ind
```

Comment:
```suggestion
                inRange *= np.logical_not(nan_ind)
```

same as above, bitwise is safe to do here, but `not` fails due to being an array. `np.logical_not` is good